pom.xml

```diff
@@ -468,39 +468,13 @@
                     <dependency>
                         <groupId>org.apache.maven.doxia</groupId>
                         <artifactId>doxia-module-markdown</artifactId>
-                        <version>1.3</version>
+                        <version>1.9.1</version>
                     </dependency>
                 </dependencies>
                 <configuration>
 
                 </configuration>
             </plugin>
-            <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <version>1.7</version>
-                <executions>
-                    <execution>
-                        <id>modify-markdown</id>
-                        <phase>pre-site</phase>
-                        <configuration>
-                            <failOnError>false</failOnError>
-                            <target>
-                                <copy todir="${project.basedir}/src/site/markdown">
-                                    <fileset dir="${project.basedir}" includes="README.md"/>
-                                </copy>
-                                <move file="${project.basedir}/src/site/markdown/README.md" tofile="${project.basedir}/src/site/markdown/manual.md"/>
-                                <replaceregexp file="${project.basedir}/src/site/markdown/manual.md"
-                                               match="```.*"
-                                               replace=""
-                                               byline="true"/>
-                            </target>
-                        </configuration>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
Nope, this way `manual.md` will never be updated again. Look at the copy task! It copies the `README.md`!

Reply:
Thank you. :+1: